committeeoversightapp/models.py

```diff
@@ -300,6 +450,39 @@ class CommitteeDetailPage(DetailPage):
         FieldPanel('hide_rating'),
     ]
 
+    def get_context(self, request):
+        context = super(CommitteeDetailPage, self).get_context(request)
+        context['latest_rating'] = context['page'].committee.latest_rating
+
+        congresses = context['page'].committee.ratings_reverse \
+            .values_list('congress_id', flat=True)
+        context['congresses'] = [x for x in congresses]
```

Comment:
Picking nits over here: Do you need to unpack this into a list, or will the template work with the output from `values_list`?

Reply:
Ditto for each of the similar lines below!

Reply:
These need to be straight lists, since they're going into my charthelper as Javascript!